### DISCUSSION TITLE
v1  - Content 📝

### DISCUSSION THREAD

**Discussion by alisonjoseph** [carbon-design-system/carbon-platform#1270] [Release feedback]
**Hi 👋  Thanks for joining our v1 "Test Fest"!** 

Below are some templates for feedback on the topic of fixes, concepts, missing content, and requests. We'd greatly appreciate if you provide suggestions to solve any issues you see. ❤️ 

```
📝 Fix
**Page:** URL
**Issue:** E.g. Change "libray" to "library"
```

```
🤓 Concept
**Page:** URL
**Issue:** E.g. This sentence has a metaphor that may be difficult for diverse audiences to understand: "Carbon is like..."
**Suggestion:** E.g. Try to use concrete examples instead of metaphors. "Carbon consists of..."
```

```
🔎 Missing content
**Page:** URL
**Issue:** E.g. The FAQ page doesn't provide answers to how Carbon is accessible
**Suggestion:** E.g. Shift content from (URL) and delete the second paragraph so that it is accurate with the new platform.
```

```
🙏 Request
**Page:** URL
**Issue:** E.g. I'm struggling to understand how maintainers can get involved with indexing. 
**Suggestion:** E.g. Maybe we need a tutorial for how maintainers can get started. It could live on the schema page.
```

**Comment by aagonzales** [comment]
🔎 Missing content
**Page:** https://next.carbondesignsystem.com/designing/figma
**Issue:** E.g. Whole page is outed, content has recently been updated to include v11 assets. Also includes separate directions for internal and external users.
**Suggestion:** E.g. Import in new markdown page from main Carbon Figma page
https://carbondesignsystem.com/designing/kits/figma

**Reply by aubrey-oneal** [reply]
Tracked here #1398

**Comment by oliviaflory** [comment]
📝 Fix
**Page:** [URL](https://next.carbondesignsystem.com/design-kits?q=)
**Issue:** Missing the Carbon for IBM.com white theme beta Figma kit, and only showing the deprecated white theme kit. 
![Screen Shot 2022-09-19 at 3 42 59 PM](https://user-images.githubusercontent.com/45692622/191102643-e4f22a5c-ea04-4c94-b688-349341aad417.png)

Figma white theme beta: https://www.figma.com/file/RB6yqMPPsGutYnGTSvwGR4/Carbon-for-IBM.com-(White-Theme)-Beta?node-id=475%3A372


**Related question:** Should we document the deprecated gray 10, gray 90 and gray 100 kits? They will eventually be replaced.

**Reply by aubrey-oneal** [reply]
Thanks for the white theme! We'll replace that link. 🙏

I think we met on this months ago and decided to document the deprecated kits and replace links as new kits are available. Let us know if you don't want them to show.

**Reply by aubrey-oneal** [reply]
Tracked here: #1395

**Comment by thyhmdo** [comment]
🙏 Request
**Page:** [URL](https://next.carbondesignsystem.com/assets/components)
**Issue:** Jumping straight into this page, it took me a few seconds to realize that I'm searching for components across IBM products.  
**Suggestion:** Maybe we can add a short description about what people can search here and how many different libraries that are available for this search. / I missed the Filter content there with all the libraries (maintainers), but it was still confusing at first coming to this page.

**Reply by aubrey-oneal** [reply]
Love it! A more guided experience is something we can revisit.

**Reply by aubrey-oneal** [reply]
Tracked here: #1396

**Comment by jeanservaas** [comment]
Not very clear that these links are behind a firewall and public peeps can't access this stuff. @aagonzales said there was a content update on Carbon recently that makes this clearer for people.

Page url: https://next.carbondesignsystem.com/designing/get-started#step-2:-access-design-tools


<img width="881" alt="image" src="https://user-images.githubusercontent.com/43144260/191106196-119179cf-6f6c-4289-adf1-b6a94c109157.png">

**Reply by aubrey-oneal** [reply]
Tracked here: #1397

**Comment by kingtraceyj** [comment]
🙏 Request - Update future case studies link
**Page:** [Case studies overview](https://next.carbondesignsystem.com/about-carbon/case-studies/overview#future-case-studies)
**Issue:** The current link in future case studies opens a default mail app. It's hard to know exactly who you are sending an email to if you don't have your mail app set up on a computer. Giving options to a user on how to contact someone would be better.
![image](https://user-images.githubusercontent.com/20582179/191106500-5a9fcf7f-484d-48fc-8eee-984c8c66c09d.png)

**Suggestion:** Maybe we should point users to the contact page or give a better way to reach us other than opening a default mail app

**Reply by aubrey-oneal** [reply]
Yeah, this can be added to the contact work.

https://github.com/carbon-design-system/carbon-platform/issues/783

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/designing/adobe-xd
**Issue:** List numbers under "Get the Adobe XD Creative Cloud libraries" all show as `1` after step 2. 

<img width="1677" alt="image" src="https://user-images.githubusercontent.com/11670886/191106962-af344926-432f-4b3c-9fa3-69bc5a04274a.png">

**Reply by aubrey-oneal** [reply]
Thanks! I think this is a bug since we are pulling from the Carbon site.

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/designing/adobe-xd
**Issue:** Update inline notification text

<img width="1674" alt="image" src="https://user-images.githubusercontent.com/11670886/191107127-e3b7e330-f8be-43c4-9c0f-5450f674bf7a.png">

**Comment by thyhmdo** [comment]
🔎 Missing content
**Page:** https://next.carbondesignsystem.com/design-kits?q=
**Issue:** Versioning numbers are missing for v10 on themes (both Carbon System and IBM Dot com)
**Suggestion:** Adding v10 numbers on the titles. For example: Figma gray 10 theme -> Figma v10 gray 10 theme 

![image](https://user-images.githubusercontent.com/77697351/191107409-45ab81e5-3d27-4e7c-b070-626aaa180536.png)

**Reply by aubrey-oneal** [reply]
Tracked here: #1399

**Comment by sstrubberg** [comment]
🔎 Missing content
**Page:** https://next.carbondesignsystem.com/about-carbon/how-carbon-works#the-carbon-ecosystem
**Issue:** The text in this section doesn't tell the same story as the graphic and the graphic *nails it*. The text does a great job of illustrating the relationship between core and local systems, but it doesn't describe our relationship with brand and IDL.
**Suggestion:** Words word words ;)

<img width="849" alt="image" src="https://user-images.githubusercontent.com/15822070/191107442-7e659b1f-0358-451e-a6ec-d540cada1cf6.png">

**Reply by aubrey-oneal** [reply]
This is also an old image: the most recent one gets rid of a typo in local systems ("for specific use cases")

**Reply by aubrey-oneal** [reply]
Tracked here: #1434

**Comment by oliviaflory** [comment]
🔎 Missing content
**Page:** https://next.carbondesignsystem.com/design-kits?q=carbon
**Issue:** If user searches "Carbon" the Carbon for IBM.com kits do not show up in the results, but other community kit assets do, like Dark theme for mobile. 
**Suggestion:** Should the Carbon for IBM.com library kits show up when user searches "Carbon"

**Reply by aubrey-oneal** [reply]
If it has Carbon in the title it should show up. Maybe this is a discussion on naming convention for the design kit.
<img width="251" alt="Screen Shot 2022-09-19 at 3 31 28 PM" src="https://user-images.githubusercontent.com/47198241/191111032-d3858b45-4f32-47cf-89bf-7408fa8e6dae.png">

**Reply by aubrey-oneal** [reply]
I see what you mean. It's odd that the mobile kit shows up even though Carbon isn't in the name. Looks like we need to revisit the search algorithm too.

**Reply by aubrey-oneal** [reply]
I think this is a bug or needs dev exploration 🐛

**Comment by jeanservaas** [comment]
Old visual content

Page url: https://next.carbondesignsystem.com/about-carbon/platform-roadmap#object-objecttarget-release:september

The v1 visual shows that the Carbon site is sunset, when it is not. We also have updated landing page designs.

<img width="1139" alt="image" src="https://user-images.githubusercontent.com/43144260/191108549-a97f4abb-077b-45a0-8240-1cd6c55fb209.png">

Same with the following images on the roadmap page that show the landing pages.

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/43144260/191108842-1088149b-fc62-4484-8eac-9f85cbd2c4bc.png">

**Reply by aubrey-oneal** [reply]
Tracked here: #1400

**Comment by laurenmrice** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/designing/get-started#step-4:-review-foundational-guidance
**Issue:** I notice the row items for `Brand` and `Brand Systems` don't include the word "IBM" in front of them for this page, but on another page ([Content guidelines](https://next.carbondesignsystem.com/guidelines/content/overview#content-foundations)) we are using the word "IBM" in front. We should probably be consistent in both places of how we want to do this.


---

**Reply by aubrey-oneal** [reply]
Good catch!

**Reply by aubrey-oneal** [reply]
Tracked here: #1435

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/elements/2x-grid/implementation#responsive-options
**Issue:** `*` in captions are being rendered as `-` in Next. 

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/11670886/191111821-afd40e0c-e962-4f5e-be25-0d3b4ca0d716.png">

**Reply by aubrey-oneal** [reply]
Need to reimport remote mdx

**Comment by laurenmrice** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/guidelines/content/overview#writing-for-accessibility
**Issue:** On the Guidelines > Content page under the Overview tab in the Writing for accessibility section, the list items need to be capitalized and end with a period. We have had a discussion recently with Michael Gower and Jan Child about these rules around lists in Gatsby.

<img width="831" alt="Screen Shot 2022-09-19 at 4 39 25 PM" src="https://user-images.githubusercontent.com/43969356/191112255-3ab3bc2c-75bd-4143-b5be-fbab38ce2ce3.png">

**Reply by laurenmrice** [reply]
Same with this page list https://carbondesignsystem.com/guidelines/content/writing-style#when-to-use-capital-letters

<img width="738" alt="Screen Shot 2022-09-19 at 5 00 33 PM" src="https://user-images.githubusercontent.com/43969356/191115589-1584998d-0c7f-4b67-8533-2ca4d140370a.png">

**Reply by aubrey-oneal** [reply]
Update on CDS site

**Comment by jeanservaas** [comment]
This relates to both the Carbon and the Platform site... I don't think we need these "Work in progress" qualifiers on the Data Viz content anymore.

Page urls:
https://next.carbondesignsystem.com/data-visualization/get-started
https://next.carbondesignsystem.com/data-visualization/chart-anatomy
https://next.carbondesignsystem.com/data-visualization/color-palettes
https://next.carbondesignsystem.com/data-visualization/legends
https://next.carbondesignsystem.com/data-visualization/axes-and-labels

* Dashboards guidance actually still is a work in progress


<img width="1168" alt="image" src="https://user-images.githubusercontent.com/43144260/191112942-0e1bb321-e1ef-48aa-9de2-00fb7bf731e3.png">

**Comment by annawen1** [comment]
📝 Fix
**Page:** [[URL](https://next.carbondesignsystem.com/libraries/ibmdotcom-web-components/latest/assets)](https://next.carbondesignsystem.com/libraries/ibmdotcom-web-components/latest/assets)
**Issue:** Universal banner status should be "Stable" and we can add the [demo link](https://www.ibm.com/standards/carbon/web-components/?path=/story/components-universal-banner--default) to its page

**Reply by annawen1** [reply]
Page: [https://next.carbondesignsystem.com/libraries/ibmdotcom-react/latest/assets](https://next.carbondesignsystem.com/libraries/ibmdotcom-react/latest/assets)

Also in the IBM.com React library assets page, the `Link list section`, `Tabs extended media`, and `Universal banner` should all be stable.

**Reply by aubrey-oneal** [reply]
Tracked here: #1436

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/elements/2x-grid/implementation#mix-and-match
**Issue:** Captions with lists are not rendering correctly

<img width="1671" alt="image" src="https://user-images.githubusercontent.com/11670886/191115302-ed32d655-7ea2-4c1c-93da-c363d23968dd.png">

**Reply by aubrey-oneal** [reply]
Dev bug 🐛

**Comment by aagonzales** [comment]
🔎 Missing content
**Page:** https://next.carbondesignsystem.com/elements/2x-grid/implementation#aspect-ratios
**Issue:** Missing inline notification 
**Suggestion:** Add info notification "Note: As of now, the 3:2 aspect ratio is not included in the grid package but there is an issue to add it."

<img width="1669" alt="image" src="https://user-images.githubusercontent.com/11670886/191116080-382f9d78-9214-4bcf-aa4d-7b0256f91421.png">

**Comment by aagonzales** [comment]
🔎 Missing content
**Page:** URL
**Issue:** 
**Suggestion:** Might want to just reimport grid page content. I know a few updates have been added and I may have missed some. 

**Missing** new "API and helpers" section
https://carbondesignsystem.com/guidelines/2x-grid/implementation#api-and-helpers

<img width="830" alt="image" src="https://user-images.githubusercontent.com/11670886/191117923-5dc7149d-8e18-4146-b5fb-b727ad086cc9.png">


**Missing** new resource tiles
https://carbondesignsystem.com/guidelines/2x-grid/implementation#resources

<img width="1669" alt="image" src="https://user-images.githubusercontent.com/11670886/191116902-f93b0510-fa91-4e1e-bd19-326ee8664f91.png">

**Comment by aagonzales** [comment]
📝 Fix
**Page:** 
https://next.carbondesignsystem.com/elements/color/overview#dark-themes
https://next.carbondesignsystem.com/elements/color/overview#light-themes
**Issue:** "Design kits" link is linking to main carbon site not Next.

**Reply by aubrey-oneal** [reply]
Tracked in #1437

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/elements/color/usage
**Issue:** Need to update content in the table on the Color/Usage page. There were several bugs in the table that have since been updated in main but I don't see the changes here.

<img width="1670" alt="image" src="https://user-images.githubusercontent.com/11670886/191120869-e87551d3-065d-4b40-9e5a-d4cfca0e3b5a.png">

**Comment by aagonzales** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/elements/icons/contribute#getting-started
**Issue:** "Icons library" Broken link in Next (it works in main carbon)

**Reply by aubrey-oneal** [reply]
Tracked in #1437

**Comment by oliviaflory** [comment]
📝 Fix
**Page:** https://next.carbondesignsystem.com/libraries/ibmdotcom-web-components and https://next.carbondesignsystem.com/libraries/ibmdotcom-react
**Issue:** "View library docs" button for Carbon for IBM.com Web Components and React results in "Page not found" 
Assuming the correct link is: https://www.ibm.com/standards/carbon/

**Reply by aubrey-oneal** [reply]
Tracked in #1437

**Comment by tomwaterton** [comment]
🤓 Concept
**Page:** (This comment is about the left-nav)
**Issue:** Some feedback/thoughts re the structure of the left-nav items.

**Background:**
I'll just start by saying that I’m a big fan of simplicity and I love how you’ve kept the left nav pretty streamlined. My suggestions here are about whether things could be streamlined even further...

Whenever I've looked at the new Carbon Platform site, the left nav seems to nicely have the following structure:
- Top half = intro / informational / reference material — that is, stuff you read once (or sometimes more) _to learn about using Carbon_.
- Bottom half = _things you use to build stuff with_.

**Suggestion 1:** 
`Icons` and `Pictograms` are currently listed under "Elements" in the top half of the nav, yet they seem to me to belong more to the bottom half (the "things you use to build stuff with" section).
While there is admittedly some guidance given in the later tabs of these two pages, first and foremost they are a library of assets for people to take and build with.

**Suggestion 2:**
In terms of the bottom half of the left nav, I wonder whether you could simplify things further by removing a couple of the dividers / sub-sections? 

To my mind, all of the items listed in the bottom half could be logically grouped under just three top-level entries:

* Assets 
    * Components
    * Data visualizations
    * Functions
    * Icons
    * Patterns
    * Pictograms
    * templates
* Design kits
* Libraries

**Suggestion 3:**
A minor one, but I wonder if tweaking “Libraries” to “Code libraries” might help people better understand what this item refers to?

**Reply by aubrey-oneal** [reply]
Thanks for your suggestions, Tom! You present some good reasoning for your suggestions. Our squad will chat about your ideas as soon as we can and I'll update you on the direction we head.
